app/torii-adapters/arcgis-oauth-bearer.js

```diff
@@ -300,6 +313,8 @@ export default EmberObject.extend({
       session.authMgr = UserSession.deserialize(sessionInfo.serializedSession);
       // remove  the prop...
       delete session.properties.serializedSession;
+    } else {
+      session.authMgr = this._createAuthManager(sessionInfo);
```

Comment:
my only concern is that after `_createAuthManager()` is called in `open()` it later calls `sessionInfo.authMgr.portal = getPortalRestUrl(portal);`, but we don't do that here in `_rehydrateSession()`.

Reply:
This is convoluted... but the short version is it's covered...
When we rehydrate the session, we still call `open` which fetches portal/self and will hit that `getPortalRestUrl(portal)`. The tricky bit was figuring out how to get the org url from the cookie/local-storage, and it turns out they stuff `urlKey` and `customBaseUrl` in there... so in `_createAuthManager` it checks for those props, and if present, uses them to cook the `portal` property... Of course Enterprise has different properties because... O_O
And if you're ever interested, I stuffed the cookie info into [the wiki](https://github.com/Esri/torii-provider-arcgis/wiki/Cookies)